schemas/ViewRoom.json

```diff
@@ -44,7 +44,12 @@
         {"const": "MobileSpaceMemberDetail", "description": "Room accessed via interacting with direct chat item in the space contact detail screen."},
         {"const": "MobileInCall", "description": "Room accessed via interacting with the incall screen."},
         {"const": "MobileSpaceMenu", "description": "Space accessed via interacting with the space menu."},
-        {"const": "MobileSpaceSettings", "description": "Space accessed via interacting with a space settings menu item."}
+        {"const": "MobileSpaceSettings", "description": "Space accessed via interacting with a space settings menu item."},
```

Comment:
Something's weird. We already triggers like `MobileSpaceSettings`

Reply:
not sure what do you mean 🤔  
The only change on this line is comma in the end, it's not a new trigger

Reply:
Sorry ...